lineage/Analyze.py

```diff
@@ -169,6 +170,7 @@ def Results(tHMMobj, pred_states_by_lineage, LL):
     results_dict["state_proportions_0"] = results_dict["state_proportions"][0]
     results_dict["accuracy_before_switching"] = 100 * np.mean(ravel_pred_states == ravel_true_states)
     results_dict["accuracy_after_switching"] = 100 * np.mean(ravel_switched_pred_states == ravel_true_states)
+    results_dict["balanced_accuracy_score"] = 100 * balanced_accuracy_score(ravel_switched_pred_states, ravel_true_states)
```

Comment:
Ah, sorry this is a really minor change but the true states should be the first argument and the estimated/predicted states should be the second argument. Feel free to do this on this branch or a new one.